networkx/algorithms/approximation/steinertree.py

```diff
@@ -0,0 +1,76 @@
+import networkx as nx
+from itertools import combinations, chain
+from networkx.utils import pairwise, not_implemented_for
+
+__all__ = ['metric_closure', 'steiner_tree']
+
+
+@not_implemented_for('directed')
+def metric_closure(G, nodes, weight="weight"):
+    """  Returns the sub-graph of the metric closure of the graph G induced by
```

Comment:
According to [PEP 257][1], the first line of a docstring should be a summary that fits within one line. Also, the phrase should be in the imperative mood. So in this case, something like `"""Return the metric closure of a graph.` would be fine. Also, subgraph does not need a hyphen.

A more detailed definition of the metric closure of a graph should follow the summary line after another blank line.

[1]: https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings